### PR TITLE
doc: add build instructions to the readme

### PR DESCRIPTION
**Pull request by lance**
This change tidies up the readme a bit, adding build instructions and information.

Signed-off-by: Lance Ball <lball@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,33 @@
+# OpenShift Serverless Previews Documentation
 
-# Knative Eventing on OpenShift Container Platform <img src="https://github.com/openshift-knative/docs/blob/master/images/knative-openshift-logo.png" width="100" height="50" />
+OpenShift Serverless Previews provides tutorials and information about
+additional, advanced use cases and integrations that are possible using
+OpenShift Container Platform and OpenShift Serverless, but are for development
+purposes only and are not officially supported by Red Hat.  This repository
+contains the source files for the OpenShift Serverless Previews documentation
+found at https://openshift-knative.github.io.
```
Comment (abrennan89) [inline README.md:8]: This link doesn't work, needs to be https://openshift-knative.github.io/docs I think

File: README.md [hunk 2]
```diff
@@ -1,7 +1,33 @@
+# OpenShift Serverless Previews Documentation
 
-# Knative Eventing on OpenShift Container Platform <img src="https://github.com/openshift-knative/docs/blob/master/images/knative-openshift-logo.png" width="100" height="50" />
+OpenShift Serverless Previews provides tutorials and information about
+additional, advanced use cases and integrations that are possible using
+OpenShift Container Platform and OpenShift Serverless, but are for development
+purposes only and are not officially supported by Red Hat.  This repository
```
Comment (abrennan89) [inline README.md:6]: I don't think this is really accurate, it's not advanced use cases or integrations, just developer preview features for OpenShift Serverless

File: README.md [hunk 3]
```diff
@@ -1,7 +1,33 @@
...
+Current projects that fall into this category include
+
+* OpenShift Serverless Functions [Developer Preview] https://openshift-knative.github.io/docs/docs/functions/about-functions.html
+
+
+## Building the Site
+
+If you would like to build this site locally, you will need to have a recent
+version of Node.js installed, as well as the site formatting tool,
+[antorra](https://docs.antora.org/antora/2.3/install/install-antora/).
```
Comment (abrennan89) [inline README.md:19]:
```suggestion
[Antora](https://docs.antora.org/antora/2.3/install/install-antora/).
```

File: README.md [hunk 4]
```diff
@@ -1,7 +1,33 @@
...
-Build and deploy Serverless applications using an event-driven infrastructure on Red Hat® OpenShift®.
+Current projects that fall into this category include
+
+* OpenShift Serverless Functions [Developer Preview] https://openshift-knative.github.io/docs/docs/functions/about-functions.html
+
+
+## Building the Site
+
+If you would like to build this site locally, you will need to have a recent
+version of Node.js installed, as well as the site formatting tool,
```
Comment (abrennan89) [inline README.md:18]: Not sure it's a "site formatting tool"? I'd just make this a bullet list instead. Also, I think Node is listed in the Antora docs as a pre-req anyway?

File: README.md [hunk 5]
```diff
@@ -1,7 +1,33 @@
...
+To build the site:
+
+```
+antora antora-playbook.yml
+```
+
+You will find the output in `build/site`.
+
+Build and deploy Serverless applications using an event-driven infrastructure
+on Red Hat® OpenShift®.
```
Comment (abrennan89) [inline README.md:30]: This couple of lines seem very random, I'd remove it and also be careful about using the ®? I've never seen us use that in docs

File: README.md [hunk 6]
```diff
@@ -1,7 +1,33 @@
...
+version of Node.js installed, as well as the site formatting tool,
+[antorra](https://docs.antora.org/antora/2.3/install/install-antora/).
+
+To build the site:
+
+```
+antora antora-playbook.yml
+```
+
+You will find the output in `build/site`.
```
Comment (abrennan89) [inline README.md:27]: I haven't done this in a minute but I'll check tomorrow or something if this is accurate. You also need to copy the files over then to replace the old ones if you actually want to publish the changes.

File: README.md (hunk 2 above)
Comment (lance) [reply README.md:6]: Fair enough. I was just copying from https://openshift-knative.github.io/docs/docs/index.html. I'll change that page as well.

File: README.md (hunk 5 above)
Comment (lance) [reply README.md:30]: I honestly have no idea where this came from. I would not have written this myself, so assume I must have copy/pasted it from somewhere. But now that you mention it, I can't find where it was. I assumed it must have been on the site, but :shrug:. Will delete.

File: README.md (hunk 6 above)
Comment (lance) [reply README.md:28]: OK - I've added those instructions, and also added a `.gitignore` file so the build directory doesn't end up getting added to the git repo

File: README.md (hunk 4 above)
Comment (lance) [reply README.md]: ... site generation tool

**Comment by lance** [comment]
@abrennan89 thanks for the review. I've incorporated your feedback, ptal.

File: README.md [hunk 7]
```diff
@@ -1,10 +1,41 @@
...
-# Knative Eventing on OpenShift Container Platform <img src="https://github.com/openshift-knative/docs/blob/master/images/knative-openshift-logo.png" width="100" height="50" />
+OpenShift Serverless Previews provides documentation, tutorials and additonal
+information about new products and features from OpenShift Serverless which are
+not officially supported by Red Hat but may be available through Developer or
+Technology Preview programs. These capabilities are provided for development
+purposes only.  This repository contains the source files for the OpenShift
+Serverless Previews documentation found at
+https://openshift-knative.github.io/docs/.
+
+Current projects that fall into this category include
```
Comment (abrennan89) [inline README.md:11]:
```suggestion
Current projects in this category include:
```

File: README.md [hunk 8]
```diff
@@ -1,10 +1,41 @@
...
+
+* OpenShift Serverless Functions [Developer Preview] https://openshift-knative.github.io/docs/docs/functions/about-functions.html
+
+
+## Building the Site
+
+If you would like to build this site locally, you will need to
+have the site generation tool,
+[Antorra](https://docs.antora.org/antora/2.3/install/install-antora/).
```
Comment (abrennan89) [inline README.md:20]:
```suggestion
[Antora](https://docs.antora.org/antora/2.3/install/install-antora/).
```

File: README.md [hunk 9]
```diff
@@ -1,10 +1,41 @@
...
+
+Current projects that fall into this category include
+
+* OpenShift Serverless Functions [Developer Preview] https://openshift-knative.github.io/docs/docs/functions/about-functions.html
+
+
+## Building the Site
+
+If you would like to build this site locally, you will need to
+have the site generation tool,
```
Comment (abrennan89) [inline README.md:19]:
```suggestion
If you want to build this site locally, you must use the site generation tool,
```

**Review by abrennan89 — CHANGES_REQUESTED** [review]
Couple more nits then lgtm - I'll check the process when I'm doing updates for the site too

**Pull request merged**